src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -357,21 +358,51 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
 
   render() {
     const { artwork, inquiryComponent } = this.props
+    const isPriceListed = !artwork?.isPriceHidden
+    const isOfferableFromInquiry = artwork?.isOfferableFromInquiry
+    const isOfferable = artwork?.is_offerable
+    const isAcquireable = artwork?.is_acquireable
+    const isInquireable = artwork?.is_inquireable
```

Comment:
We can probably remove the aliases on the graphql query, and just destruct the artwork object, no?
`const { isOfferable, isAcquireable, isInquireable } = artwok`

Reply:
This comp is pretty old...was there a reason they were aliased in the first place? I was hesitant to make this code change without understanding the reason for it to begin with so I just limited changes to the scope of the ticket.